app/src/main/java/com/github/multimatum_team/multimatum/JsonDeadlineConverter.kt

```diff
@@ -0,0 +1,20 @@
+package com.github.multimatum_team.multimatum
+
+import com.google.gson.GsonBuilder
+import java.time.LocalDateTime
+import com.github.multimatum_team.multimatum.model.Deadline
+
+class JsonDeadlineConverter {
+    val gsonBuilder = GsonBuilder().registerTypeAdapter(LocalDateTime::class.java,
+        LocalDateTimeSerializer()).registerTypeAdapter(LocalDateTime::class.java,
+        LocalDateTimeDeserializer())
+    val gson = gsonBuilder.create()
+
+    fun toJson(deadline: Deadline): String? {
```

Comment:
You could use the `fun toJson(deadline: Deadline): String? = ...` syntax. Similar for the function below.